tests/test_06_l2vpn_return_codes.py

```diff
@@ -95,6 +93,7 @@ def test_013_create_l2vpn_with_vlan_untagged_code201(self):
         response = requests.post(api_url, json=payload)
         assert response.status_code == 201, response.text
 
+    @pytest.mark.xfail(reason="return status 400 -> Validation error: Scheduling not possible: 2025-01-22 16:47:13.554115+00:00 start_time cannot be before the current time\nScheduling not possible: 2025-12-31T12:00:00Z end_time is not in a valid ISO format")
```

Comment:
Related to [#381](https://github.com/atlanticwave-sdx/sdx-controller/issues/381)
Related to[#288](https://github.com/atlanticwave-sdx/sdx-controller/issues/388)